libs/application/templates/marriage-conditions/src/lib/constants.ts

```diff
@@ -0,0 +1,97 @@
+import { DefaultEvents } from '@island.is/application/types'
+
+export type Events =
+  | { type: DefaultEvents.SUBMIT }
+  | { type: DefaultEvents.PAYMENT }
+  | { type: DefaultEvents.ASSIGN }
+
+export enum States {
+  DRAFT = 'draft',
+  PAYMENT = 'payment',
+  SPOUSE_CONFIRM = 'spouse_confirm',
+  DONE = 'done',
+}
+export enum Roles {
+  APPLICANT = 'applicant',
+  ASSIGNED_SPOUSE = 'assigned_spouse',
+}
+
+export const YES = 'yes'
+export const NO = 'no'
+
+export enum ApiActions {
+  submitApplication = 'submitApplication',
+  createCharge = 'createCharge',
+  assignSpouse = 'assignSpouse',
+}
+
+export enum MarriageTermination {
+  divorce = 'divorce',
+  lostSpouse = 'lostSpouse',
+  annulment = 'annulment',
+}
+
+export type DistrictCommissionerAgencies = {
+  name: string
+  place: string
+  address: string
+  id: string
+}
+
+export const ReligiousLifeViewingSocieties = [
```

Comment:
Not a review comment but just curious. Is this really not accessible from an API?